extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -110,6 +118,12 @@ const renderDefaultPageAttachmentUI = (pageEl, attachmentEl) => {
 const renderPageAttachmentUiWithImages = (win, pageEl, attachmentEl) => {
   const openAttachmentEl = buildOpenInlineAttachmentElement(pageEl);
 
+  // Setting theme
+  const theme = attachmentEl.getAttribute('theme');
+  if (theme && AttachmentTheme.DARK === theme.toLowerCase()) {
+    openAttachmentEl.setAttribute('theme', 'dark');
```

Comment:
nit:
Would it be good to reference the same string for the theme value?
openAttachmentEl.setAttribute('theme', AttachmentTheme.DARK);

Reply:
makes sense, fixed